src/main/resources/publishAndTag

```diff
@@ -22,11 +22,33 @@ if [[
 
   # Only tag on first travis job
   if [[ "${TRAVIS_JOB_NUMBER##*.}" == "1" ]]; then
-    git config --global user.email "builds@travis-ci.com"
-    git config --global user.name "Travis CI"
+    HEAD_SHA=$(git rev-parse HEAD)
```

Comment:
Adequate permission was proving illusive so switched to github api and access token based tagging.